components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -4586,6 +4587,20 @@ public static Cache getAPIContextCache() {
         return tenantDomains;
     }
 
+    public static Set<String> getTenantDomainsByState(String state) throws UserStoreException {
```

Comment:
Shall we add method comments

Reply:
fixed https://github.com/wso2/carbon-apimgt/pull/6811/commits/a840167e827f482f100d25dadea5b4a46dc53be1